src/legacy/ui/public/index_patterns/index.ts

```diff
@@ -60,3 +58,6 @@ export {
   IndexPatterns,
   StaticIndexPattern,
 } from '../../../core_plugins/data/public';
+
+// stubs
+export { stubFields, stubIndexPattern } from '../../../core_plugins/data/public/stubs';
```

Comment:
Please remove all test code from index.ts and other non-test files

Reply:
Separated